lb.tf

```diff
@@ -1,5 +1,5 @@
 module "lb" {
-  source = "git::https://github.com/appuio/terraform-modules.git//modules/vshn-lbaas-cloudscale?ref=v2.7.0"
+  source = "git::https://github.com/appuio/terraform-modules.git//modules/vshn-lbaas-cloudscale?ref=v3.0.0"
```

Comment:
Sorry to be picky, but is lbaas update directly related to the Terraform update?

Reply:
Can we rephrase the PR title to be more specific "Bump Terraform to v1.3.x" or "Bump Terraform to v1.3.x and module lbaas-cloudscale to v3.0.0".

Reply:
Those changes belong together because that update also removes the experimental changes :)

Renamed PR.